modules/services/mako.nix

```diff
@@ -5,371 +5,79 @@
-        type = types.nullOr types.bool;
-        description = ''
-          Whether or not to show icons in notifications.
-        '';
-      };
-
-      maxIconSize = mkOption {
-        default = 64;
-        type = types.nullOr types.int;
-        description = ''
-          Set maximum icon size to the specified number of pixels.
-        '';
-      };
-
-      iconPath = mkOption {
-        default = null;
-        type = types.nullOr types.str;
-        description = ''
-          Paths to search for icons when a notification specifies a name
-          instead of a full path. Colon-delimited. This approximates the search
-          algorithm used by the XDG Icon Theme Specification, but does not
-          support any of the theme metadata. Therefore, if you want to search
-          parent themes, you'll need to add them to the path manually.
-
-          The {file}`/usr/share/icons/hicolor` and
-          {file}`/usr/share/pixmaps` directories are
-          always searched.
-        '';
-      };
-
-      markup = mkOption {
-        default = true;
-        type = types.nullOr types.bool;
-        description = ''
-          If 1, enable Pango markup. If 0, disable Pango markup. If enabled,
-          Pango markup will be interpreted in your format specifier and in the
-          body of notifications.
-        '';
-      };
-
-      actions = mkOption {
-        default = true;
-        type = types.nullOr types.bool;
-        description = ''
-          Applications may request an action to be associated with activating a
-          notification. Disabling this will cause mako to ignore these requests.
-        '';
-      };
-
-      format = mkOption {
-        default = "<b>%s</b>\\n%b";
-        type = types.nullOr types.str;
-        description = ''
-          Set notification format string to format. See FORMAT SPECIFIERS for
-          more information. To change this for grouped notifications, set it
-          within a grouped criteria.
-        '';
-      };
-
-      defaultTimeout = mkOption {
-        default = 0;
-        type = types.nullOr types.int;
-        description = ''
-          Set the default timeout to timeout in milliseconds. To disable the
-          timeout, set it to zero.
-        '';
-      };
-
-      ignoreTimeout = mkOption {
-        default = false;
-        type = types.nullOr types.bool;
-        description = ''
-          If set, mako will ignore the expire timeout sent by notifications
-          and use the one provided by default-timeout instead.
-        '';
-      };
-
-      groupBy = mkOption {
-        default = null;
-        type = types.nullOr types.str;
-        description = ''
-          A comma-separated list of criteria fields that will be compared to
-          other visible notifications to determine if this one should form a
-          group with them. All listed criteria must be exactly equal for two
-          notifications to group.
-        '';
-      };
```

Comment:
Use `mkRenamedOptionModule` to redirect the old options to the new settings option path.